Can't publish android target locally when android sdk preview version is installed

If I publish android target on my m1 macbook I get an error. But if I run it on a linux system it works. Tried both project's android sdk (via local.properties) and intellij's android sdk, same result. What am I doing wrong? 🤔 

```sh
./gradlew :skiko:publishToMavenLocal \
    -Pdeploy.version=0.8.9.1 \
    -Pskiko.awt.enabled=true \
    -Pskiko.js.enabled=true \
    -Pskiko.wasm.enabled=true \
    -Pskiko.android.enabled=true \
    -Pskiko.native.enabled=true \
    -Pskiko.native.linux.enabled=true \
    -Pskiko.native.mac.enabled=true \
    -Pskiko.native.ios.enabled=true \
    -Pskiko.native.ios.arm64.enabled=true \
    -Pskiko.native.ios.simulatorArm64.enabled=true \
    -Pskiko.native.ios.x64.enabled=true
```

error:
```sh
> Task :skiko:compileKotlinAndroid FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':skiko:compileKotlinAndroid'.
> Could not resolve all files for configuration ':skiko:androidCompileClasspath'.
   > For input string: "VanillaIceCream"
```

Using intellij:
```
IntelliJ IDEA 2024.1.5 (Community Edition)
Build #IC-241.18968.26, built on August 4, 2024
Runtime version: 17.0.11+1-b1207.30 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.5
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 10
Metal Rendering is ON
Registry:
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
Non-Bundled Plugins:
  org.plugin.dot.id (1.5.4)
  com.jetbrains.space (241.18968.26)
  Lombook Plugin (241.18034.62)
  org.jetbrains.idea.grammar (2022.3.2)
  com.jetbrains.packagesearch.intellij-plugin (241.0.11)
  org.jetbrains.android (241.17011.79)
  com.squareup.sqldelight (2.0.2)
  androidx.compose.plugins.idea (241.17011.79)
  org.jetbrains.compose.desktop.ide (1.6.2)
  com.suusan2go.kotlin-fill-class (1.0.23)
Kotlin: 241.18968.26-IJ
```

Not sure if relevant, but `androidMain` sources seems detached(?)
![image](https://github.com/user-attachments/assets/5a978186-180d-48f2-a1f5-245b5c7cb7fa)

--- a/skiko/buildSrc/src/main/kotlin/tasks/configuration/JvmTasksConfiguration.kt
+++ b/skiko/buildSrc/src/main/kotlin/tasks/configuration/JvmTasksConfiguration.kt
@@ -541,7 +541,7 @@ fun Project.androidJar(askedVersion: String = ""): Provider<File> =
             val platformsDir = androidHome.resolve("platforms")
             val versions = platformsDir.list().orEmpty()
             versions.maxByOrNull { name -> // possible name: "android-32", "android-33-ext4"
-                name.split("-").getOrNull(1)?.toInt() ?: 0
+                name.split("-").getOrNull(1)?.toIntOrNull() ?: 0
             } ?: error(
                 buildString {
                     appendLine("'$platformsDir' does not contain any directories matching expected 'android-NUMBER' format: ${versions}")

